src/database/Database.cpp

```diff
@@ -119,7 +119,7 @@ Database::applySchemaUpgrade(unsigned long vers)
 
     case 4:
         BanManager::dropAll(*this);
-        DataFrame::dropAll(*this); // not and error, it changed again
+        DataFrame::addLastModifiedField(*this);
```

Comment:
What do you think about just making this: 
 mSession << "ALTER TABLE accountdata ADD lastmodified INT NOT NULL DEFAULT 0;"
Seems more straightforward than scattering it around in DataFrame since the SQL is only used under these conditions.